src/main/java/io/spokestack/spokestack/tts/SSML.java

```diff
@@ -0,0 +1,84 @@
+package io.spokestack.spokestack.tts;
+
+/**
+ * <p>
+ * A simple wrapper class for an SSML string.
+ * </p>
+ *
+ * <p>
+ * <a href="https://www.w3.org/TR/speech-synthesis11/">SSML</a>
```

Comment:
I will be borrowing this for the iOS docs, thanks for writing it up!

Reply:
👍 had to be done sometime. It's going somewhere on the site too.